src/cider/exec/nextgen/jitlib/llvmjit/LLVMJITUtils.h

```diff
@@ -99,6 +99,15 @@ inline llvm::Value* getLLVMConstantFP(double value,
       return nullptr;
   }
 }
+
+inline llvm::Value* getLLVMConstantGlobalStr(const std::string& value,
+                                             llvm::IRBuilder<>* builder,
+                                             llvm::LLVMContext& ctx) {
+  llvm::Value* str_lv = builder->CreateGlobalString(
+      value, "str_const_" + std::to_string(std::hash<std::string>()(value)));
+  str_lv = builder->CreateBitCast(str_lv, getLLVMPtrType(JITTypeTag::INT8, ctx));
+  return str_lv;
```

Comment:
Do we need length? or it ends with `\0`.
Or we always need to get length info from expression since it's a constant expression.

Reply:
The value returned here does not contain length. If length is needed, it can be acquired from the `std::string` input (for C++), or from the `JITExprValue` of the returned constant expression (for LLVM IR).

BTW, seems the global string is nul-terminated, [doc](https://llvm.org/doxygen/classllvm_1_1IRBuilderBase.html#a3a7e4483b56895f73ad5898251dc4b4c).